app/models/file_upload.rb

```diff
@@ -0,0 +1,5 @@
+class FileUpload < ActiveRecord::Base
+  include ImageUploader::Attachment(:image)
```

Comment:
Is a pdf considered an image? Has this been tested with pdfs?

Reply:
Yes and yes